src/views/docs-view/utils/__tests__/get-back-to-link.test.ts

```diff
@@ -0,0 +1,99 @@
+import { ProductName, ProductSlug } from 'types/products'
+import { getBackToLink } from '../get-back-to-link'
+
+describe('fullPathFromRelativeHref', () => {
```

Comment:
Little typo here!

Reply:
Ah thanks, renamed the function and forgot to rename it here 😅 Will handle in my next PR! 👍